spec/features/saml/loa1_sso_spec.rb

```diff
@@ -139,16 +132,15 @@
   end
 
   context 'visiting IdP via SP, then going back to SP and visiting IdP again' do
-    it 'maintains the request_id in the params' do
+    it 'displays the branded page' do
       authn_request = auth_request.create(saml_settings)
       visit authn_request
-      sp_request_id = ServiceProviderRequest.last.uuid
 
-      expect(current_url).to eq sign_up_start_url(request_id: sp_request_id)
+      expect(current_url).to match(%r{/http:\/\/www\.example\.com\/sign_up\/start\?request_id=.+/})
 
       visit authn_request
 
-      expect(current_url).to eq sign_up_start_url(request_id: sp_request_id)
+      expect(current_url).to match(%r{/http:\/\/www\.example\.com\/sign_up\/start\?request_id=.+/})
```

Comment:
When using `%r{` no need to pad with or escape `/` characters, I think ths needs to be:

```ruby
expect(current_url).to match(%r{http://www\.example\.com/sign_up/start\?request_id=.+})
```